CLI.md

```diff
@@ -281,7 +284,7 @@ Options:
   -id, --rule-id TEXT
   -r, --replace-id           Replace rule IDs with new IDs before export
   -h, --help                 Show this message and exit.
-(detection-rules-build) (base) ➜  detection-rules git:(rule-loader) ✗
+(detection-rules-build) (base) ➜  detection-rules git:(main) ✗
 ```
 
 Alternatively, rules can be exported into a consolidated ndjson file which can be imported in the Kibana security app
```

Comment:
Please add sub-header to highlight this command.

Reply:
Would it make sense to move the upload-rule command below and make it more clear not to use it? Is it removed in 9.0?

Reply:
++

Reply:
Agreed! Updated.